DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingUsingFunctions/projectingUsingFunctions.java

```diff
@@ -0,0 +1,45 @@
+//region Usings
+import net.ravendb.client.documents.session.IDocumentSession;
+import net.ravendb.client.documents.session.IRawDocumentQuery;
+//endregion
+import net.ravendb.demo.common.DocumentStoreHolder;
+import java.util.List;
+
+public class projectingUsingFunctions {
```

Comment:
uppercase for java classes (both class and file name)

Reply:
Done

Reply:
Locally .java has changed but git didn't got the change